app/sidekiq/central_mail/submit_form4142_job.rb

```diff
@@ -131,20 +136,39 @@ def lighthouse_service
       @lighthouse_service ||= BenefitsIntakeService::Service.new(with_upload_location: true)
     end
 
-    def upload_to_lighthouse
-      Rails.logger.info(
-        'Successful Form4142 Submission to Lighthouse',
-        { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
-      )
-
-      payload = {
-        upload_url: lighthouse_service.location,
+    def payload_hash(lighthouse_service_location)
+      {
+        upload_url: lighthouse_service_location,
         file: { file: @pdf_path, file_name: @pdf_path.split('/').last },
         metadata: generate_metadata.to_json,
         attachments: []
       }
+    end
 
+    def upload_to_lighthouse
+      Rails.logger.info(
+        'Successful Form4142 Upload Intake UUID aquired from Lighthouse',
+        { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
+      )
+
+      payload = payload_hash(lighthouse_service.location)
       lighthouse_service.upload_doc(**payload)
+
+      log_info = { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
```

Comment:
Do you want to move this to the top of the method and replace the hash on L151?

Reply:
yes, yes I do. Fixed in c81a8601f2